appveyor.yml

```diff
@@ -16,7 +16,7 @@ build_script:
   - cmake --version
   # The build script starts in root.
   - set JSONCPP_FOLDER=%cd%
-  - set JSONCPP_BUILD_FOLDER=%JSONCPP_FOLDER%\build\release
+  - set JSONCPP_BUILD_FOLDER=%JSONCPP_FOLDER%\build_out\release
```

Comment:
Why `build_out` for **cmake**? Is that to be consistent with a bagel folder?

Reply:
> Name conflict. The CI has been building in `build` folder, but now `BUILD` is a file in the repo.

Anyway, I've removed this change and renamed the Bazel file to `BUILD.bazel`. This name is less popular, but officialy recognized by Bazel.